worker/src/RTC/PipeTransport.cpp

```diff
@@ -588,25 +588,31 @@ namespace RTC
 				  packet->GetPayloadType(),
 				  packet->GetSequenceNumber());
 
+				// Remove this SSRC.
+				RecvStreamClosed(packet->GetSsrc());
```

Comment:
I don't think this is needed. If `DecryptSrtp()` fails ther won't be any new SSRC stream into libsrtp, I checked this by looking at the libsrtp code.